styleguide/derek/_partials/grid/widget/blog-feed-item.ejs

```diff
@@ -0,0 +1,23 @@
+<!-- Blog Feed Pattern -->
+
+<div class="blogFeed-item">
+  <div class="row">
```

Comment:
This helps prevent other conflicts with something else that may come along the way.  `.row` is such a generic class name, we're just asking for some crappy module plugin to break things.